Orange/tests/test_knn.py

```diff
@@ -7,23 +7,27 @@
 
 from Orange.data import Table, Domain, ContinuousVariable, DiscreteVariable
 from Orange.classification import KNNLearner
-from Orange.evaluation import CA, CrossValidation
+from Orange.regression import KNNRegressionLearner
+from Orange.evaluation import CA, CrossValidation, MSE
 
 
 class TestKNNLearner(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         cls.iris = Table('iris')
-        cls.learn = KNNLearner()
+        cls.learn = KNNLearner
```

Comment:
I would skip assigning classes in to cls.learn and cls.learn_reg and just use their names in tests.

self.learn() is not much shorter than KNNLearner() (in fact it has the same number of characters)